CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 Changes
 =======
 
+Version 0.9.2 (UNRELEASED)
+--------------------------
+
+- Fixes job status consumer not to fail while fetching workflow engine logs of workflows that couldn't be scheduled.
```

Comment:
What about:

- Fixes job status consumer exception while attempting to fetch workflow engine logs for workflows could not have been successfully scheduled.